crates/symbolicator/src/service.rs

```diff
@@ -442,18 +414,27 @@ impl RequestService {
                     sentry::end_session_with_status(SessionStatus::Exited);
                     SymbolicationResponse::Completed(Box::new(response))
                 }
-                Err(error) => {
+                Ok(Err(err)) => {
+                    // NOTE: This will double-report this error to Sentry.
+                    // We want the `tracing::error` here for local command line output, but we also
+                    // want the `capture_anyhow` which will give a full error with a symbolicated backtrace.
+                    // So until we can actually get a `Backtrace` out of an `Error`, and `anyhow`
+                    // implements that functionality through its `Error` impl, we are stuck with this for now.
+                    // See <https://github.com/rust-lang/rust/issues/99301>
+                    let error: &dyn std::error::Error = err.as_ref();
+                    tracing::error!(error, "Symbolication failed");
+                    sentry_anyhow::capture_anyhow(&err);
```

Comment:
this is unfortunate indeed. Ideally I would want to only keep the `tracing::error`, but we are not there yet.